<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form4</class>
 <widget class="QWidget" name="Form4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库连接参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>address.svg</normaloff>address.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form4</receiver>
   <slot>continueClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form4</receiver>
   <slot>cancelClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>continueClick()</slot>
  <slot>cancelClick()</slot>
 </slots>
</ui>
